### PR TITLE
Add ifx bug with associate-stmt

### PR DESCRIPTION
**Pull request by ktras**

### CODE REVIEW

File: compiler/Fortran/ifx/associate-stmt/mod_m.mod
Comment (everythingfunctional) [inline compiler/Fortran/ifx/associate-stmt/mod_m.mod:1]: You should remove this file.

**Review by everythingfunctional — APPROVED** [review]
Can confirm that I get that error with the given example, and that I believe it is a compiler bug.

**Comment by rouson** [comment]
@everythingfunctional please add me as a contributor to this repo so that I can review this PR if that's ok.

File: compiler/Fortran/ifx/associate-stmt/mod_m.mod
Comment (ktras) [reply compiler/Fortran/ifx/associate-stmt/mod_m.mod:1]: Thanks for catching that mistake. I didn't even notice!!

**Comment by ktras** [comment]
> Can confirm that I get that error with the given example, and that I believe it is a compiler bug.

@everythingfunctional Thank you for confirming. I have simplified the reproducer in `example.f90` even more, so you may want to check again to make sure the error still occurs on a machine other than the one I am using.

**Comment by everythingfunctional** [comment]
> @everythingfunctional please add me as a contributor to this repo so that I can review this PR if that's ok.

I sent you an invitation a day or two ago

**Comment by everythingfunctional** [comment]
Confirmed. LGTM.

**Pull request merged**